tests/fixtures.ts

```diff
@@ -124,12 +121,22 @@ export const test = baseTest.extend<TestFixtures & TestOptions, WorkerFixtures>(
     browserProcess?.kill();
   },
 
+  rewriteFile: async ({ mcpMode }, use) => {
+    await use((file: string) => {
+      if (mcpMode === 'docker')
+        return path.relative(path.dirname(test.info().config.configFile!), file);
```

Comment:
I think this surfaces a real issue - users that run Playwright MCP server in Docker don't expect the files to be generated in Docker. I wonder if we should disable all the file saving in that mode and expose those files as resources in the future.

What I'm suggesting: do not run tests that require this rebase since this is not what user expects.